eZ/Bundle/EzPublishRestBundle/EzPublishRestBundle.php

```diff
@@ -18,6 +19,11 @@ public function build(ContainerBuilder $container)
         $container->addCompilerPass(new Compiler\OutputVisitorPass());
         $container->addCompilerPass(new Compiler\ValueObjectVisitorPass());
 
+        /** @var \eZ\Bundle\EzPublishCoreBundle\DependencyInjection\EzPublishCoreExtension $eZExtension */
+        $eZExtension = $container->getExtension('ezpublish');
+        $eZExtension->addConfigParser(new ConfigParser\RestResources());
+        $eZExtension->addDefaultSettings(__DIR__ . '/Resources/config', ['default_settings.yml']);
```

Comment:
Why was loading the `default_settings.yml` file moved here? I don't think there's a reason for that.

I suspect this is the reason why the tests are not loading the config.